geo/src/algorithm/minimun_rotated_rect.rs

```diff
@@ -0,0 +1,106 @@
+use crate::{
+    algorithm::{centroid::Centroid, rotate::Rotate, BoundingRect, CoordsIter},
+    coords_iter, Area, ConvexHull, CoordFloat, GeoFloat, GeoNum, Point, Polygon,
+};
+/// Returns the minimun bounding rect of polygon.
+///
+
+pub trait MinimunRotatedRect<'a, T> {
```

Comment:
It looks like `T` is unused in the definition.  Remove?

Reply:
i found `ConvexHull` function has same usage for `T`，not sure about its real meaning.

Reply:
if i delete `T`,it will have lifetime error

Reply:
Oh lol, let me create a separate issue to track removing it there.  Let's leave this as-is for now.

Reply:
any process with this?